src/sentry/api/serializers/models/group.py

```diff
@@ -141,7 +141,8 @@ def get_attrs(self, item_list, user):
             ).select_related('user')
         )
 
-        user_counts = tagstore.get_group_values_seen([g.id for g in item_list], 'sentry:user')
+        user_counts = tagstore.get_group_values_seen(
```

Comment:
I think we are going to want to be able to make the serializers also environment aware at some point (at least this was my plan for the TSDB changes.) For example, if you are viewing the stream filtered to production, stats should probably reflect production.

I don't know as this needs to be done as part of this change (probably not) but I think it's worth keeping in mind for later.

Reply:
Ya, good call, I can take a wack at that in another PR unless you beat me to it.